modules/deploy/partials/kubernetes/guides/start-streaming.adoc

```diff
@@ -11,18 +11,102 @@ alias rpk-topic="kubectl --namespace <namespace> exec -i -t redpanda-0 -c redpan
 
 . Create a topic called `twitch_chat`:
 +
+[tabs]
+======
+Helm + Operator::
++
+--
+
+.. Create a xref:manage:kubernetes/manage-topics.adoc[Topic resource]:
++
+.`topic.yaml`
+[,yaml]
+----
+apiVersion: cluster.redpanda.com/v1alpha1
+kind: Topic
+metadata:
+  name: twitch_chat
```

Comment:
```suggestion
  name: twitch-chat
```

The problem is
```
kubectl apply -f topic-twitch-chat.yaml
The Topic "twitch_chat" is invalid: metadata.name: Invalid value: "twitch_chat": a lowercase RFC 1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*')
```

Reply:
This seems like another limitation we should document. cc @mattschumpert